.gitignore

```diff
@@ -10,3 +10,4 @@ rs.cred
 *.h5
 *.dat
 tests/run_inputs.py
+cyclus.sqlite
```

Comment:
This seems correct but unrelated - is it intentional?  Maybe because it's correct it doesn't matter...